accounts/abi/method.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
-// Callable method given a `Name` and whether the method is a constant.
+// Method represents a method ABI entry.
```

Comment:
Not really. Look at the whole comment. I think it's more clear now. The `Name` is part of it's public interface, and other part of the comment mentions about constant part.

Reply:
No it definitely drops critical information regarding the ABI as the field "constant" is a core component of the current ABI. This will eventually be changed to "stateMutability" but for now it is needed, especially if the goal is to reduce breaking changes.

Reply:
Well, the second line tells:
```
If the method is `Const` no transaction needs to be created for this
```
I don't want to insist here. I will update to:
```
Method represents a callable given a `Name` and whether the method is a constant.
```